### PR TITLE
Percentage methods tt

### PR DESCRIPTION
**Pull request by tylertomlinson**
Percentage methods from Iteration 1.

### CODE REVIEW

File: lib/game_collection.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 require_relative "game"
 require "csv"
+require "pry"
```
Comment (mronauli) [inline lib/game_collection.rb]: should we have the require "pry" in our master?

Comment (d-atkins) [inline lib/game_collection.rb]: I would take the require pry out

File: lib/game_collection.rb [hunk 2]
```diff
@@ -13,4 +13,15 @@ def create_games(file_path)
...
+  def percentage_home_wins
+    home_wins = @games.find_all do |game|
+      game.home_goals > game.away_goals
+    end
+    (home_wins.length.to_f / @games.length).round(2)
+  end
+
+  
+
+
```
Comment (Yetidancer) [inline lib/game_collection.rb]: Is this where the end was messing things up

File: lib/game_collection.rb [hunk 3]
```diff
@@ -14,24 +15,44 @@ def create_games(file_path)
...
+    end
+    (visitor_wins.to_f / @games.length).round(2)
+  end
+
+  def percentage_ties
+    ties_count = @games.count do |game|
+      game.home_goals == game.away_goals
+    end
+    (ties_count.to_f / @games.length).round(2)
+  end
```
Comment (mronauli) [inline lib/game_collection.rb:36]: looks good!

File: test/game_collection_test.rb [hunk 4]
```diff
@@ -1,5 +1,7 @@
 require './test/test_helper'
 require './lib/game_collection'
+require './lib/game'
+require './lib/game_teams'
```
Comment (d-atkins) [inline test/game_collection_test.rb]: I think you can take this out?

File: lib/game_collection.rb [hunk 5]
```diff
@@ -29,6 +29,11 @@ def percentage_visitor_wins
     (visitor_wins.to_f / @games.length).round(2)
   end
 
-
+  def percentage_visitor_wins
+    visitor_wins = @games.count do |game|
+      game.away_goals > game.home_goals
+    end
+    (visitor_wins.to_f / @games.length).round(2)
+  end
```
Comment (Yetidancer) [inline lib/game_collection.rb]: Different than how I envisioned (using game_teams) but it passes

**Review by d-atkins — APPROVED** [review]
I think it's ready

File: lib/game_collection.rb (hunk 2 above)
Comment (tylertomlinson) [reply lib/game_collection.rb]: Yes. Fixed!

File: lib/game_collection.rb (hunk 5 above)
Comment (tylertomlinson) [reply lib/game_collection.rb]: THANKS ZEKE

File: lib/game_collection.rb (hunk 1 above)
Comment (tylertomlinson) [reply lib/game_collection.rb]: No I took it out. My bad

Comment (tylertomlinson) [reply lib/game_collection.rb]: On it!

File: test/game_collection_test.rb (hunk 4 above)
Comment (tylertomlinson) [reply test/game_collection_test.rb]: Got it!

**Review by tylertomlinson — COMMENTED** [review]
Ready to be merge to master now.

**Pull request merged**